src/jdk.internal.vm.ci/share/classes/jdk.vm.ci.hotspot/src/jdk/vm/ci/hotspot/HotSpotConstantPool.java

```diff
@@ -928,9 +930,9 @@ public void loadReferencedType(int cpi, int opcode, boolean initialize) {
 
                 break;
             case "InvokeDynamic":
-                if (isInvokedynamicIndex(cpi)) {
+                /*if (isInvokedynamicIndex(cpi)) {
                     compilerToVM().resolveInvokeDynamicInPool(this, cpi);
-                }
+                }*/
```

Comment:
Is there something to fix here?

Reply:
That's a vestige of old code that I don't believe is necessary anymore. Invokedynamic is resolved further up so that can be removed. I think it makes sense to leave the invokedynamic case for completeness, but it will be left blank.